doc/UsersGuide/source/conf.py

```diff
@@ -24,7 +24,7 @@
 author = ' '
 
 # The short X.Y version
-version = 'develop'
+version = 'v2.0.0'
```

Comment:
@jdkublnick For now, I think we should change this back to `develop` because it's going into the `develop` branch/version of the documentation. I know it's describing HAFS v2, but unless it's the production/hafs.v2 branch, we should probably keep this. That said, we still need to use a substitution for the latest release, so you could add the following lines and update the substitution you added earlier: 
```
# Documentation-wide substitutions

rst_prolog = """
.. |latestr| replace:: v2.0.0
```